src/git/zcl_abapgit_git_transport.clas.abap

```diff
@@ -4,18 +4,19 @@ CLASS zcl_abapgit_git_transport DEFINITION
   CREATE PUBLIC .
 
   PUBLIC SECTION.
+    TYPES: ty_deepen_level TYPE n LENGTH 2.
```

Comment:
suggest being able to pass any number, there is no need to add restrictions(eg. passing 100 as deepen level wont work), or adding extra TYPES